.github/workflows/test.yml

```diff
@@ -20,5 +20,5 @@ jobs:
   pip:
     uses: ./.github/workflows/test_template.yml
     with:
-      runs-on: '["ubuntu-latest", "macos-latest", "windows-latest"]'
+      runs-on: '["ubuntu-latest", "macos-latest", "windows-latest", "macos-14"]'
       python-version: '["3.9", "3.10", 3.11, 3.12]'
```

Comment:
Looks like there are being captured correctly, but for the sake of consistency, it might be good to add double quotes for 3.11 an 3.12.

Reply:
done